code/modules/events/event_container.dm

```diff
@@ -176,6 +176,7 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Drone Uprising",						/datum/event/rogue_maint_drones,		25,		list(ASSIGNMENT_ENGINEER = 30)),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Supermatter Power Surge",				/datum/event/power_surge,				100,	list(ASSIGNMENT_ENGINEER = 10)),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Bluespace Drive Instability",			/datum/event/bsd_instability,			50),
+		new /datum/event_meta(EVENT_LEVEL_MODERATE, "DeepmaintRooms",						/datum/event/backrooms_deepmaint,		50),
```

Comment:
Should this really be a moderate event? I've not poked the event code much so please do correct any misconceptions, but when this event ends, aren't all people stuck in deepmaint just... SOL and trapped forever?

Reply:
They aren't, generally. Even during the event, you can escape. This is not a property of the event, just of Deepmaint itself. 

All standard Deepmaint turfs have a small chance of having a hidden station teleport trait, which itself has a probability of triggering on each crossing.

Why the double-probability? Well, for one, it keeps the number of rolls down, but mostly - this is the main actual _threat_. Not monsters or other hostiles (even though they may rarely be added eventually) - there is a small-moderate chance you get walled in a section with no station teleporters in it.

Even then, if you die, you will get yeeted to regular maint.

Reply:
Oh. DERP. Just saw that it deletes all the _entrances_. My brain misread this as all teleporters for wfc.